lib/fizzy/capi.cpp

```diff
@@ -7,10 +7,79 @@
 #include "instantiate.hpp"
 #include "parser.hpp"
 #include <fizzy/fizzy.h>
+#include <cstring>
 #include <memory>
 
 namespace
 {
+inline void set_success(FizzyError* error) noexcept
+{
+    if (error == nullptr)
+        return;
+
+    error->code = FIZZY_SUCCESS;
+    error->message[0] = '\0';
+}
+
+template <size_t DestSize>
+inline size_t strlcpy_s(char* dest, const char* src) noexcept
```

Comment:
Would probably just call this `safe_strlcpy` and leave a comment mentioning this is inspired by `strlcpy_s`.

Reply:
Renamed.